docs/onboarding/VPN.md

```diff
@@ -13,20 +24,39 @@ Address = X.X.X.X/32
 DNS = 10.20.50.1
 
 [Peer]
-PublicKey = [VPNServerPublicKey]
+PublicKey = Gh7UqjaA2IzrcB9ZNiFco7zVpwHmCeFCHLuny/UETw0=
 PresharedKey = [presharedKey]
 AllowedIPs = 10.20.50.0/24, X.X.X.X/32
 Endpoint = boulder.vpn.ethpandaops.io:51820
 PersistentKeepalive = 15
 ```
 
-Note: `VPNServerPublicKey` and `presharedKey` and `X` address will be provided to you in order to authenticate your device.
+Note:  `X` address and `presharedKey` will be provided to you in order to authenticate your device. `youPrivateKey` is the key generated in earlier step 1. (Related to the shared `pubkey`).
+
+#### Client configuration for Berlin:
+1. Create a wireguard.conf file following the wireguard-client-config below
+2. Provide the public key to Barnabas#1342/parithosh#0530 on discord or anyone from the [ethpandaops team](https://github.com/orgs/ethpandaops/people).
+
+```
+[Interface]
+PrivateKey = [yourPrivateKey]
+Address = X.X.X.X/32
+
+[Peer]
+PublicKey = Z4iicKvlim92Et+xxjYAD54rJ4DcYhdoeuZoxAfTCTA=
+PresharedKey = [presharedKey]
+AllowedIPs = 10.10.42.1/24,10.10.11.1/24, X.X.X.X/32
+Endpoint = berlin.vpn.ethpandaops.io:51820
```

Comment:
maybe we can also list here boulder.vpn..

Reply:
wdym? you can't connect to boulder.vpn if the config is for berlin

Reply:
me idiot.